File: real estate web/BE/src/main/java/com/c0722g1repobe/dto/post/create_post/CreatePostDto.java
package com.c0722g1repobe.dto.post.create_post;

import lombok.*;

import java.util.List;

@Builder
@Setter
@AllArgsConstructor
@NoArgsConstructor
@Getter
public class CreatePostDto {
    private Long idCustomer;
    private String namePost;
    private Long idDemand;
    private Long idLandType;
    private Long idWards;
    private Long idDirection;
    private String numberAddress;
    private Double price;
    private Double area;
    private String note;
    private String[] imageListURL;
}

File: real estate web/FE/src/app/dto/post/post-create/create-post-dto.ts
export interface CreatePostDto {
  idCustomer: number;
  namePost: string;
  idDemand: number;
  idLandType: number;
  idWards: number;
  idDirection: number;
  numberAddress: string;
  price: number;
  area: number;
  note?: string;
  imageListURL: string[];
}
